packages/frontend/src/pages/NewInstructLabSession.svelte

```diff
@@ -0,0 +1,255 @@
+}
+function removeSkills(file: string): void {
+  skillsFiles = skillsFiles.toSpliced(skillsFiles.indexOf(file), 1);
+}
+
+async function addKnowledge(): Promise<void> {
+  const files = await requestExplorerModal('Select knowledge');
+  knowledgeFiles.push(...files.map(file => file.path));
+}
+
+function setTrainingType(type: 'knowledge' | 'skills'): void {
+  switch (trainingType) {
+    case 'knowledge':
+      skillsFiles = [];
+      break;
+    case 'skills':
+      knowledgeFiles = [];
+      break;
+  }
+  trainingType = type;
+}
+
+function openInstructLabDocumentation(): void {
+  studioClient
+    .openURL(
+      'https://github.com/instructlab/instructlab?tab=readme-ov-file#-creating-new-knowledge-or-skills-and-training-the-model',
+    )
+    .catch((err: unknown) => {
+      console.error(err);
+    });
+}
+</script>
+
+<FormPage
+  title="New InstructLab Session"
+  breadcrumbLeftPart="InstructLab Sessions"
+  breadcrumbRightPart="New session"
+  breadcrumbTitle="Go back to sessions"
+  onclose={goToUpPage}
+  onbreadcrumbClick={goToUpPage}>
+  <svelte:fragment slot="icon">
+    <div class="rounded-full w-8 h-8 flex items-center justify-center">
+      <Fa size="1.125x" class="text-[var(--pd-content-header-icon)]" icon={faPlus} />
+    </div>
+  </svelte:fragment>
+  <svelte:fragment slot="content">
+    <div class="flex flex-col w-full">
+      <!-- form -->
+      <div class="bg-[var(--pd-content-card-bg)] m-5 space-y-6 px-8 sm:pb-6 xl:pb-8 rounded-lg h-fit">
+        <div class="w-full flex flex-col gap-y-4">
+          <!-- model input -->
+          <div>
+            <label for="model" class="pt-4 block mb-2 font-bold text-[var(--pd-content-card-header-text)]"
+              >Model to Fine Tune</label>
+            <ModelSelect models={$modelsInfo} disabled={false} bind:value={model} />
+          </div>
+
+          <!-- session name -->
+          <div>
+            <label for="session-name" class="block mb-2 font-bold text-[var(--pd-content-card-header-text)]"
+              >Session name</label>
+            <Input bind:value={sessionName} class="grow" name="session-name" aria-label="session name" />
+            <span class="text-[var(--pd-table-body-text)]"
+              >Name of the session to be able to easily find it in your list of sessions.</span>
+          </div>
+
+          <!-- file(s) input -->
+          <div class="flex flex-col gap-y-1">
+            <!-- knowledge -->
+            <div
+              class:border-2={true}
+              class="rounded-md p-5 cursor-pointer bg-[var(--pd-content-card-inset-bg)] flex flex-col gap-y-2"
+              aria-label="Knowledge"
+              aria-pressed={trainingType === 'knowledge' ? 'true' : 'false'}
+              class:border-[var(--pd-content-card-border-selected)]={trainingType === 'knowledge'}
+              class:border-[var(--pd-content-card-border)]={trainingType !== 'knowledge'}>
+              <button
```

Comment:
Why Button is not used ?

Reply:
We don't have backend so no submit see https://github.com/containers/podman-desktop-extension-ai-lab/pull/1849#discussion_r1795073662